components/filter-leads-card/index.tsx

```diff
@@ -138,30 +138,53 @@ const FilterLeadsCard: React.FC<FilterLeadsCardProps> = ({
           <div className='border-t my-[10px] w-[349px] h-[1px] border-[#E9F0FF]'></div>
 
           {/* Date Range */}
-          <div className='my-2'>
-            <label className='font-semibold text-[#00156A] text-[16px] mb-1'>
+          <div className='my-[10px]'>
+            <label className='font-600 leading-[28px] text-[#00156A] text-[16px] mb-1'>
               Date Range
             </label>
             <div className='bg-white w-full mt-2 text-center'>
-              <DateRangePicker
-                startDate={startDate}
-                startDateId='your_unique_start_date_id'
-                endDate={endDate}
-                endDateId='your_unique_end_date_id'
-                onDatesChange={({ startDate, endDate }) => {
-                  setStartDate(startDate);
-                  setEndDate(endDate);
-                }}
-                focusedInput={focusedInput}
-                onFocusChange={(focusedInput) => setFocusedInput(focusedInput)}
-                displayFormat={() => 'MMM D, YYYY'}
-                anchorDirection='left'
-              />
+              <div className='flex space-x-2 items-center'>
+                <div className='relative w-[159px] h-[44px]'>
+                  <div className='py-2 px-2 border border-gray-300 rounded-md focus:outline-none focus:border-blue-500 date-style flex justify-start'>
+                    <DatePicker
+                      selected={startDate}
+                      onChange={(date: any) => setStartDate(date)}
+                      selectsStart
+                      startDate={startDate}
+                      endDate={endDate}
+                      dateFormat='MMM d, yyyy'
+                    />
+                  </div>
+                  <span className='absolute right-2 top-1/2 transform -translate-y-1/2 text-gray-600'>
+                    <CalendarIcon />
+                  </span>
+                </div>
+                <span className='text-[#667085]'>–</span>
+                <div className='relative w-[159px] h-[44px]'>
+                  <div className='py-2 px-2 border border-gray-300 rounded-md focus:outline-none focus:border-blue-500 date-style flex justify-start'>
+                    <DatePicker
+                      selected={endDate}
+                      onChange={(date: any) => setEndDate(date)}
+                      selectsEnd
+                      startDate={startDate}
+                      endDate={endDate}
+                      minDate={startDate}
+                      // withPortal
```

Comment:
If this and following commented codes are not necessary then remove them completely.